04 - Guides, Workflows, & Courses/Guides/How to get the most out of the Breadcrumbs plugin.md

```diff
@@ -0,0 +1,78 @@
+
+---
+aliases: 
+- 
+tags:
+- evergreen
+publish: true
+---
+
+The [[breadcrumbs|Breadcrumbs]] is a very powerful plugin. As such, it seems to have a rather steep learning curve. However, after some initial set-up, you can already start reaping the benefits of arbitrary, non-exclusive hierarchies within your vault. 
+
+*Note: this is a refined version of the post by [[lkadre]] on [this discussion](https://github.com/SkepticMystic/breadcrumbs/discussions/175) on Github.*
+
+#### Setting your index
+
+In the Trail/Grid settings, you can set a page to act as your Index. In my case, my Index is my homepage. It's the first page people see when they visit my Publish website and it's where they go back to. 
```

Comment:
Is your publish page public? If so, I think it might be super helpful to link to it here, as I think if people see an example of what this all looks like, it might help them have a better understanding.

Reply:
It should be linked on the forum, but same as with the others, I'd rather @lkadre add it if she feels comfortable with it.